src/Microsoft.DocAsCode.Dfm/Rules/DfmVideoBlockRule.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Dfm
+{
+    using System.Text.RegularExpressions;
+
+    using Microsoft.DocAsCode.MarkdownLite;
+
+    public class DfmVideoBlockRule : IMarkdownRule
+    {
+        private static readonly Regex _videoRegex = new Regex(@"^ *\[\!Video +(?<link>https?\:\/\/.+?) *\] *(\n|$)", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
>^ *\[\!Video +(?<link>https?\:\/\/.+?) *\] *(\n|$) [](start = 64, length = 50)

Question: is it possible use syntax like GFM? [https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#youtube-videos](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#youtube-videos)